reco/LST1_Hillas.py

```diff
@@ -210,26 +210,22 @@ def guess_type(filename):
             l = np.rad2deg(np.arctan2(hillas.length, foclen))
 
             #Calculate Timing parameters
-#            peak_time = units.Quantity(peakpos[chan])*units.Unit("ns")
-#            timepars = time.timing_parameters(geom.pix_x,geom.pix_y,clean,peak_time,hillas.psi)
-            pedcorrectedsamples[~window] = 0.0            
-            pedcorrectedsamples[pedcorrectedsamples < 0] = 0.001 # to avoid sum=0
-            ind = np.indices(data.shape)[2]
-            wmtime = np.average(ind, axis=2, weights=pedcorrectedsamples) # weighted mean time
- 
-            wmtime_ns = units.Quantity(wmtime[0])*units.Unit("ns")
-            timegrad, intcpt = time.timing_parameters(geom.pix_x,geom.pix_y,clean,wmtime_ns,hillas.psi)
-
-            # Calculate Time Gradient from shower head            
-            if np.sign(hillas.skewness) != 0 :
-                timegrad *= (-1.0) * np.sign(hillas.skewness)
-            print('time gradient : ',timegrad)
+            peak_time = units.Quantity(peakpos[chan])*units.Unit("ns")
+            timepars = time.timing_parameters(geom,clean,peak_time,hillas)
 
             # Calculate Leakage parameters
             LeakParas = leakage(geom, clean, cleanmask)
             leak1 = LeakParas['leakage1_intensity']
             leak2 = LeakParas['leakage2_intensity']
             
+            # Calculate Impact parameter
```

Comment:
can this calculation go to a different function so we only need to call it from the main script (LST1_Hillas.py)? It would also be nice if you could also write test modules for these functions (see ctapipe test modules to see examples).